techniques/general/G9.html

```diff
@@ -51,102 +49,89 @@
 
 <div class="default-grid with-gap leftcol">
 <aside class="box nav-hack sidebar standalone-resource__sidebar">
-    <header class="box-h">Page Contents</header>
+    <header class="box-h">このページの内容</header>
     <div class="box-i">
       <nav aria-label="page contents" class="navtoc">
-        <ul><li><a href="#technique">About this Technique</a></li>
-<li><a href="#description">Description</a></li>
-<li><a href="#examples">Examples</a></li>
-<li><a href="#related">Related Techniques</a></li>
-<li><a href="#tests">Tests</a></li>
+        <ul><li><a href="#technique">このテクニックについて</a></li>
+<li><a href="#description">解説</a></li>
+<li><a href="#examples">事例</a></li>
+<li><a href="#related">関連テクニック</a></li>
+<li><a href="#tests">検証</a></li>
 </ul>
       </nav>
     </div>
 </aside>
 
 <main id="main" class="standalone-resource__main">
-<!-- 日本語訳されるまでの一時的な注記 -->
-<div class="untranslated-note"><p>このWCAG 2.2 テクニック集の日本語訳は作業中となっています。WCAG 2.1 達成方法集の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Techniques/">WCAG 2.1 達成方法集</a> ]</p></div>
-<h1><span>Technique G9:</span>Creating captions for live synchronized media</h1>
+
+<h1><span>テクニック G9:</span>ライブの同期したメディアに対してキャプションを作成する</h1>
 
 
 
 
 <section id="technique" class="box">
-	<h2 class="box-h box-h-icon">About this Technique</h2>
+	<h2 class="box-h box-h-icon">このテクニックについて</h2>
 	<div class="box-i">
 
 
   
-  <p>This technique relates to:</p>
+  <p>このテクニックは、次に関連する:</p>
   <ul>
   
     <li>
       
 <a href="https://w3c.github.io/wcag/understanding/captions-live">1.2.4: Captions (Live)</a>
-(<strong>Sufficient</strong>, together with <a href="../general/G93">G93: Providing open (always visible) captions</a>)
+(<a href="../general/G93">G93: Providing open (always visible) captions</a> とともに<strong>十分なテクニック</strong>)
     </li>
   
     <li>
       
 <a href="https://w3c.github.io/wcag/understanding/captions-live">1.2.4: Captions (Live)</a>
-(<strong>Sufficient</strong>, together with <a href="../general/G87">G87: Providing closed captions</a>)
+(<a href="../general/G87">G87: Providing closed captions</a> とともに<strong>十分なテクニック</strong>)
     </li>
   
     <li>
       
 <a href="https://w3c.github.io/wcag/understanding/captions-live">1.2.4: Captions (Live)</a>
-(<strong>Sufficient</strong>, together with <a href="../general/G87">G87: Providing closed captions</a>
-  using a more specific technique)
+(より具体的なテクニックを用いれば <a href="../general/G87">G87: Providing closed captions</a> とともに<strong>十分なテクニック</strong>)
```

Comment:
Sufficient using a more specific technique の訳ですが、G131 https://github.com/waic/wcag22/pull/1197 と合わせて「より詳細なテクニックを用いれば十分なテクニック」とするのはいかがでしょうか。
```suggestion
(より詳細なテクニックを用いれば <a href="../general/G87">G87: Providing closed captions</a> とともに<strong>十分なテクニック</strong>)
```